### PR TITLE
Bug 1802664: *: fix etcdmemberscontroller

### PR DESCRIPTION
**Pull request by alaypatel07**
picked up the changes missed from #139

### CODE REVIEW

File: pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go [hunk 1]
```diff
@@ -129,44 +110,44 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 		}))
 		if updateErr != nil {
 			c.eventRecorder.Warning("EtcdMembersErrorUpdatingStatus", updateErr.Error())
-			return err
+			return updateErr
```
Comment (deads2k) [inline pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go]: don't return these errors, just keep going.

Comment (deads2k) [reply pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go]: you may aggregate them if you wish, but set everything

**Comment by deads2k** [comment]
/lgtm
/hold 

holding for comment

File: pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go (hunk 1 above)
Comment (alaypatel07) [reply pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go]: done

**Pull request merged**